investos/portfolio/risk_model/factor_risk.py

```diff
@@ -1,23 +1,351 @@
 import cvxpy as cvx
+import forecastos as fos  # from pypi
 import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+from scipy.stats import mstats
+from sklearn.preprocessing import StandardScaler
 
 import investos.util as util
 from investos.portfolio.risk_model import BaseRisk
+from investos.portfolio.risk_model.utils.risk_adjustments import (
+    AssetDigonalVarAdjuster,
+    FactorCovAdjuster,
+)
+
+fos_return_uuid_dict = {
+    # For momentum and market beta
```

Comment:
Let's remove comments since they can be wrong (like for this one)

Reply:
(Remove comments for return_uuid_dict)

Reply:
Hmm actually some are helpful. I'll mark which ones to remove. This is one to remove.